gymnasium/envs/phys2d/conversion.py

```diff
@@ -0,0 +1,77 @@
+from typing import Any, Dict, Optional, Tuple
+
+import jax.random as jrng
+
+import gymnasium as gym
+from gymnasium import Space
+from gymnasium.envs.registration import EnvSpec
+from gymnasium.functional import ActType, FuncEnv, StateType
+
+
+class JaxEnv(gym.Env):
+    """
+    A conversion layer for numpy-based environments.
+    """
+
+    state: StateType
+    rng: jrng.PRNGKey
+
+    def __init__(
+        self,
+        func_env: FuncEnv,
+        observation_space: Space,
+        action_space: Space,
+        metadata: Optional[Dict[str, Any]] = None,
+        render_mode: Optional[str] = None,
+        reward_range: Tuple[float, float] = (-float("inf"), float("inf")),
+        spec: Optional[EnvSpec] = None,
+    ):
+        """Initialize the environment from a FuncEnv."""
+        if metadata is None:
+            metadata = {}
+        self.func_env = func_env
+        self.observation_space = observation_space
+        self.action_space = action_space
+        self.metadata = metadata
+        self.render_mode = render_mode
+        self.reward_range = reward_range
+        self.spec = spec
+
+        if self.render_mode == "rgb_array":
+            self.render_state = self.func_env.render_init()
+        else:
+            self.render_state = None
+
+    def reset(self, *, seed: Optional[int] = None, options: Optional[dict] = None):
+        super().reset(seed=seed)
+        if not hasattr(self, "rng") or seed is not None:
```

Comment:
I think we should initialise "rng" in `__init__` with `seeding.np_random()` using the second value which is the random seed value. 
Then we only update seed when `seed is not None`

Reply:
Added